src/components/Literals/Scalar/PrimitiveValue.tsx

```diff
@@ -10,7 +10,7 @@ function primitiveToString(primitive: Primitive): string {
         case 'boolean':
             return !!primitive.boolean ? 'true' : 'false';
         case 'datetime':
-            return formatDate(timestampToDate(primitive.datetime!));
+            return formatDateUTC(timestampToDate(primitive.datetime!));
```

Comment:
These are the dates that show in the "Inputs" tab for a task/execution?

Reply:
The `Literals` components are used anywhere we render a flyte `Literal` value to the screen. So that would be the inputs/outputs modal for workflow executions, the inputs/outputs tab in the details panel for a task execution, and I think maybe certain cases in the Launch form when rendering a fixed input value that the user cannot change.